Makefile

```diff
@@ -413,6 +414,14 @@ images: image bundle-image  ## Build operator and bundle images.
 .PHONY: images-extra
 images-extra: openscap-image e2e-content-images  ## Build the openscap and test content images.
 
+.PHONY: build-osde2e-test-binary
+build-osde2e-test-binary:
+	$(GO) test ./tests/e2e/rosa -c -v -o rosa.test
```

Comment:
This should build the test binary. At this point we can build the container image and give it a shot in a ROSA environment using the osde2e tools.

https://github.com/openshift/osde2e/tree/main#tests